src/subdomains/generic/user/models/bank-data/bank-data.entity.ts

```diff
@@ -20,6 +21,10 @@ export enum BankDataVerificationError {
 }
 
 @Entity()
+@Index((bankData: BankData) => [bankData.iban, bankData.userData], {
+  unique: true,
+  where: `type = '${BankDataType.USER}'`,
+})
```

Comment:
Does this interfere with current data?

Reply:
Jap that should not be a problem 👍  I checked it manually in prd DB. Looks good 👍